File: PurpleBalls.Data/Models/Player.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PurpleBalls.Data.Models
{
    public class Player
    {
        public int PlayerId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: PurpleBalls/ClientApp/src/models/Player.ts
export class Player {
    constructor(
        public playerId: number,
        public firstName: string,
        public lastName: string,
        public email: string,
        public phoneNumber: string,
        public isDeleted: boolean,
        public shirtNumber: number,
        public shirtName: string,
    ) { }
  }
